### PR TITLE
Update run-best-practices-analyzer-scans-and-manage-scan-results.md

### PR DESCRIPTION
**Pull request by doctordns**
Added later server versions on which BPA runs to the list of Windows Server versions supported. Added Server 2016, 2019.
Simplified references to Windows server later in the document.
Made some minor layout and grammar changes.

### CODE REVIEW

**Comment by PRMerger14** [comment]
@doctordns : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/administration/server-manager/run-best-practices-analyzer-scans-and-manage-scan-results.md [hunk 1]
```diff
@@ -14,22 +14,22 @@ ms.date: 10/16/2017
...
 
-Best Practices Analyzer (BPA) is a server management tool that is available in Windows Server 2012 R2,  Windows Server 2012, and  Windows Server 2008 R2. BPA can help administrators reduce best practice violations by scanning roles that are installed on managed servers that are running  Windows Server 2012  or  Windows Server 2008 R2, and reporting best practice violations to the administrator.
+Best Practices Analyzer (BPA) is a server management tool that is available in Windows Server 2019, Windows Server 2016, Windows Server 2012 R2,  Windows Server 2012, and  Windows Server 2008 R2. BPA can help administrators reduce best practice violations by scanning roles that are installed on managed servers that are running a supported version of, and reporting best practice violations to the administrator.
```
Comment (illfated) [inline WindowsServerDocs/administration/server-manager/run-best-practices-analyzer-scans-and-manage-scan-results.md:17]: I am fairly certain we can simplify the sentence further by skipping "that is", which is an overused "filler" expression in the English language.
```suggestion
Best Practices Analyzer (BPA) is a server management tool available in Windows Server 2019, Windows Server 2016, Windows Server 2012 R2,  Windows Server 2012, and  Windows Server 2008 R2. BPA can help administrators reduce best practice violations by scanning roles that are installed on managed servers that are running a supported version of, and reporting best practice violations to the administrator.
```

**Comment by illfated** [comment]
Other than that, I approve of the already suggested changes in this PR.

File: WindowsServerDocs/administration/server-manager/run-best-practices-analyzer-scans-and-manage-scan-results.md (hunk 1 above)
Comment (doctordns) [reply WindowsServerDocs/administration/server-manager/run-best-practices-analyzer-scans-and-manage-scan-results.md:17]: Looks Fine to me

File: WindowsServerDocs/administration/server-manager/run-best-practices-analyzer-scans-and-manage-scan-results.md [hunk 2]
```diff
@@ -57,9 +57,9 @@ The following table describes the best practice rules categories against which r
...
+In Windows Server, some roles prompt you to specify additional parameters, such as the names of specific servers or shares that are running parts of the role, or the IDs of submodels, before starting a BPA scan. For BPA scans on models that require you to specify additional parameters, use the BPA cmdlets; the BPA GUI cannot accept additional parameters such as submodel IDs. For example, the submodel ID **FSRM** represents the File Services BPA submodel for File Server Resource Manager, a role service of File and Storage Services. To run a scan only on the File Server Resource Manager role service, run a BPA scan by using Windows PowerShell cmdlets, and add the parameter `SubmodelId` to your cmdlet.
 
-Although you cannot pass additional parameters to a scan that you start in the BPA GUI, the BPA tile in Server Manager displays results for the most recent BPA scan, regardless of how the scan was started.
+Although you cannot pass additional parameters to a scan that you start in the BPA GUI, the BPA tile in Server Manager displays results for the most recent BPA scan, regardless of how you started the scand.
```
Comment (illfated) [inline WindowsServerDocs/administration/server-manager/run-best-practices-analyzer-scans-and-manage-scan-results.md:62]: Oh. I almost missed this one, a typo:
```suggestion
Although you cannot pass additional parameters to a scan that you start in the BPA GUI, the BPA tile in Server Manager displays results for the most recent BPA scan, regardless of how you started the scan.
```

**Comment by PRMerger20** [comment]
@doctordns : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/administration/server-manager/run-best-practices-analyzer-scans-and-manage-scan-results.md (hunk 2 above)
Comment (doctordns) [reply WindowsServerDocs/administration/server-manager/run-best-practices-analyzer-scans-and-manage-scan-results.md]: good catch

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger8** [comment]
@illfated: I'm sorry - only the author of this article, @JasonGerend, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by illfated** [comment]
Yes, I know. My sign-off comment is in agreement with the team members.

**Comment by PRMerger15** [comment]
@sethmanheim : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger15** [comment]
@sethmanheim : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**